docs/parsers/typescript.md

```diff
@@ -23,35 +27,32 @@ import eslintPluginBetterTailwindcss from "eslint-plugin-better-tailwindcss";
 
 export default [
   {
-    files: ["**/*.{ts,tsx,cts,mts}"],
-    languageOptions: {
-      parser: eslintParserTypeScript,
-      parserOptions: {
-        project: true
-      }
-    }
-  },
-  {
-    plugins: {
-      "better-tailwindcss": eslintPluginBetterTailwindcss
-    },
-    rules: {
-      // enable all recommended rules to report a warning
-      ...eslintPluginBetterTailwindcss.configs["recommended-warn"].rules,
-      // enable all recommended rules to report an error
-      ...eslintPluginBetterTailwindcss.configs["recommended-error"].rules,
-
-      // or configure rules individually
-      "better-tailwindcss/enforce-consistent-line-wrapping": ["warn", { printWidth: 100 }]
-    },
+    // enable all recommended rules
+    ...eslintPluginBetterTailwindcss.configs.recommended,
+
+    // override rules to configure them individually
+    // rules: {
+    //   "better-tailwindcss/enforce-consistent-line-wrapping": ["warn", { printWidth: 100 }]
+    // },
+
     settings: {
       "better-tailwindcss": {
         // tailwindcss 4: the path to the entry file of the css based tailwind config (eg: `src/global.css`)
         entryPoint: "src/global.css",
         // tailwindcss 3: the path to the tailwind config file (eg: `tailwind.config.js`)
         tailwindConfig: "tailwind.config.js"
       }
+    },
+
+    files: ["**/*.{ts,tsx,cts,mts}"],
+
+    languageOptions: {
+      parser: eslintParserTypeScript,
```

Comment:
According to the latest typescript-eslint docs, the parser is added slightly differently:

```ts
import tseslint from 'typescript-eslint';

...
    {
      languageOptions: {
        parser: tseslint.parser,
        ...
    },
```

It is only necessary to install `typescript-eslint` dependency. I haven't used legacy eslintrc config so haven't checked the latest recommended settings for it. There may be a change there too.